docs/tutorials/generic_field_type/4_introduce_a_template.md

```diff
@@ -0,0 +1,41 @@
+# Step 4 - Introduce a template
+
+!!! tip
+
+    You can find all files used and modified in this step on [GitHub]().
+
+## Point 2D template
+
+In order to display data from the Field Type, you need to create and register a template for it.
+Each Field Type template receives a set of variables that can be used to achieve the desired goal.
+In this case the most important variable is the `field`, an instance of `eZ\Publish\API\Repository\Values\Content\Field`.
+In addition to its own metadata (`id`, `fieldDefIdentifier`, etc.), it exposes the Field Value through the `value` property.
+
+Remember that [Field Type templates can be overridden](../../guide/twig_functions_reference.md#override-a-field-template-block) in order to tweak what is displayed and how.
+For more information, see the documentation about [Field Type templates](../../api/field_type_form_and_template.md#content-view-templates).
+
+First, create a `point2d_field.html.twig` template in the `templates` directory.
+It will define the default display of a Point 2D.
+Your basic template for Point 2D should look like this:
+
+```html+twig
+{% block point2d_field %}
+    ({{ field.value.getX() }}, {{ field.value.getY() }})
+{% endblock %}
+```
+
+The `field.value.getX` and  the `field.value.getY` are piped through the `raw` Twig operator, since the variable contains HTML code.
+Without it, the HTML markup would be visible directly, because Twig escapes variables by default.
```

Comment:
Sorry, but in the code snippet above, there is not `raw` Twig operator.  What you are referring too?